platooning_strategic_IHP/src/platoon_manager_ihp.cpp

```diff
@@ -631,4 +631,63 @@ namespace platoon_strategic_ihp
         // ---> 4.4 return IHP calculated desired speed
         return pos_des;
     }
+
+    // UCLA: find the clsest index to the cut-in joining vehicle.
+    // Note: The index pointing to the gap-leading vehicle's index.
```

Comment:
This Note is not clear at all.  It has no verb.  It seems to say that an index is pointing to an index.